src/MessageProducer.php

```diff
@@ -1,26 +1,25 @@
 <?php
 /*
- * This file is part of the codeliner/psb-bernard-dispatcher.
- * (c) Alexander Miertsch <kontakt@codeliner.ws>
+ * This file is part of the prooph/psb-bernard-dispatcher.
+ * (c) 2014 - 2015 prooph software GmbH <contact@prooph.de>
  *
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
- * 
- * Date: 31.10.14 - 15:08
+ *
+ * Date: 10/31/14 - 03:08 PM
  */
 namespace Prooph\ServiceBus\Message\Bernard;
 
 use Bernard\Producer;
-use Prooph\Common\Messaging\RemoteMessage;
-use Prooph\ServiceBus\Message\RemoteMessageDispatcher;
+use Prooph\Common\Messaging\Message;
 
 /**
- * Class MessageDispatcher
+ * Class MessageProducer
  *
  * @package Prooph\ServiceBus\Message\Bernard
  * @author Alexander Miertsch <kontakt@codeliner.ws>
  */
-class MessageDispatcher implements RemoteMessageDispatcher
+class MessageProducer
```

Comment:
Agreed. The interface doesn't influence the new dispatching logic of prooph/service-bus but make it more explicit how a message producer should look like.

I will add the interface to prooph/service-bus.